### PR TITLE
Add 3 pixels to the guide star selection edge padding

### PR DESCRIPTION
**Pull request by jeanconn**
Add 3 pixels to the guide star selection edge padding

### CODE REVIEW

File: proseco/guide.py [hunk 1]
```diff
@@ -331,8 +331,10 @@ def get_initial_guide_candidates(self):
         # Add a filter for stars that are too close to the chip edge including dither
         r_dith_pad = self.dither.row
         c_dith_pad = self.dither.col
-        row_max = CCD['row_max'] - (CCD['row_pad'] + CCD['window_pad'] + r_dith_pad)
-        col_max = CCD['col_max'] - (CCD['col_pad'] + CCD['window_pad'] + c_dith_pad)
+        row_max = (CCD['row_max'] -
```
[the remark this answers is not in the document]
Comment (jeanconn) [reply proseco/guide.py]: Sure.  I think this was all in `ok` and then `ok` got broken up by some of the include_ids work and we meant to get back to some reunifcation.

[the remark this answers is not in the document]
Comment (jeanconn) [reply proseco/guide.py]: I moved `ok` down which somewhat keeps things together.  Not sure if we need to move 'offchip' and 'imp_mag' tasks to a different subroutine for this (wrt to increasing footprint as you say).

**Comment by jeanconn** [comment]
The different question is if we want to include the stars in the 3pix region in the candidates table (so they will go through the other checks).  I don't know if it would be helpful to have debug information on them if we get desperate when looking for stars and trying to add one manually.

If we want to go that way, it seems like we'd leave the out-of-bounds check limits the way they were and then exclude the stars near the edge with a new check + logging of rejection reason etc.

**Comment by taldcroft** [comment]
For now keep it simple.  I almost even regret making the comment that resulted in the last commit.  This patch needs to be pretty tight.  And it should be against 4.3.x as a bug fix.  We can do something similar against master but are a little less constrained.

**Comment by taldcroft** [comment]
> ... if we want to include the stars in the 3pix region in the candidates table

With a broader perspective now, I think the answer is a firm no.  If there is a problem then the intent is for roll optimization to pick them up.

**Comment by jeanconn** [comment]
I just double-checked that this actually passes tests (it had been a while) and I think this is good.

**Pull request merged**